Autopilot/Inc/TelemPathInterface.hpp

```diff
@@ -25,7 +25,7 @@ struct Telemetry_Waypoint_Data_t {
 
 
 //Data for path manager to send to telemetry.
-typedef struct POGI{
+typedef struct Telemetry_POGI_t{
```

Comment:
no need for the typedef:

```cpp
struct Telemetry_POGI_t {

};
```

If you do need the typedef cause of compiler issues (shouldn't happen), the syntax is this:

```cpp
typedef struct {

} Telemetry_POGI_t;
```